### PR TITLE
[Docs] update users and roles section

### PR DESCRIPTION
**Pull request by mcassier31**
- Unify the whole content following technical writing guidelines.
- Add an "Add a new user/role" section with screenshots

### CODE REVIEW

File: doc/22_Administration_of_Pimcore/07_Users_and_Roles.md [hunk 1]
```diff
@@ -55,62 +63,48 @@ The following list outlines what the different system permissions (available for
...
 ![User permission workspaces](../img/permissions1.png)
 
 The user permissions on element basis are summed up as follows:
 
-* **list**: element can be listed in tree
-* **view**: element can be opened
-* **save**: element can be saved (save button visible)
-* **publish**: element can be published (publish button visible)
-* **unpublish**: element can be unpublished (unpublish button visible); does not exist for assets
+* **list**: elements can be listed in tree
```
Comment (fashxp) [inline doc/22_Administration_of_Pimcore/07_Users_and_Roles.md]: these permissions really are for that particular element ... so we should leave that with `element`

**Review by fashxp — CHANGES_REQUESTED** [review]

File: doc/22_Administration_of_Pimcore/07_Users_and_Roles.md (hunk 1 above)
Comment (mcassier31) [reply doc/22_Administration_of_Pimcore/07_Users_and_Roles.md]: I changed that :)

File: doc/22_Administration_of_Pimcore/07_Users_and_Roles.md [hunk 2]
```diff
@@ -55,62 +63,48 @@ The following list outlines what the different system permissions (available for
...
 ![User permission workspaces](../img/permissions1.png)
 
 The user permissions on element basis are summed up as follows:
 
-* **list**: element can be listed in tree
-* **view**: element can be opened
-* **save**: element can be saved (save button visible)
-* **publish**: element can be published (publish button visible)
-* **unpublish**: element can be unpublished (unpublish button visible); does not exist for assets
+* **list**: this element can be listed in tree
```
Comment (fashxp) [inline doc/22_Administration_of_Pimcore/07_Users_and_Roles.md]:
```suggestion
* **list**: this element (and potential child elements) can be listed in tree
```

File: doc/22_Administration_of_Pimcore/07_Users_and_Roles.md [hunk 3]
```diff
@@ -55,62 +63,48 @@ The following list outlines what the different system permissions (available for
...
+* **view**: this element can be opened
```
Comment (fashxp) [inline doc/22_Administration_of_Pimcore/07_Users_and_Roles.md]:
```suggestion
* **view**: this element (and potential child elements) can be opened
```

File: doc/22_Administration_of_Pimcore/07_Users_and_Roles.md [hunk 4]
```diff
@@ -55,62 +63,48 @@ The following list outlines what the different system permissions (available for
...
+* **save**: this element can be saved (save button visible)
```
Comment (fashxp) [inline doc/22_Administration_of_Pimcore/07_Users_and_Roles.md]:
```suggestion
* **save**: this element (and potential child elements) can be saved (save button visible)
```

File: doc/22_Administration_of_Pimcore/07_Users_and_Roles.md [hunk 5]
```diff
@@ -55,62 +63,48 @@ The following list outlines what the different system permissions (available for
...
+* **publish**: this element can be published (publish button visible)
```
Comment (fashxp) [inline doc/22_Administration_of_Pimcore/07_Users_and_Roles.md]:
```suggestion
* **publish**: this element (and potential child elements) can be published (publish button visible)
```

File: doc/22_Administration_of_Pimcore/07_Users_and_Roles.md [hunk 6]
```diff
@@ -55,62 +63,48 @@ The following list outlines what the different system permissions (available for
...
+* **unpublish**: this element can be unpublished (unpublish button visible); does not exist for assets
```
Comment (fashxp) [inline doc/22_Administration_of_Pimcore/07_Users_and_Roles.md]:
```suggestion
* **unpublish**: this element (and potential child elements) can be unpublished (unpublish button visible); does not exist for assets
```

File: doc/22_Administration_of_Pimcore/07_Users_and_Roles.md [hunk 7]
```diff
@@ -55,62 +63,48 @@ The following list outlines what the different system permissions (available for
...
 * **create**: new child elements can be created (does not exist for assets)
-* **delete**: element can be deleted
-* **rename**: name of the element can be changed
-* **settings**: element's settings can be managed i.e. the settings tab is visible; the settings permission also the path and thereby the right to move the element in tree
-* **versions**: versions tab available
-* **properties**: properties tab available and can be managed
-
-An individual user is granted access to all defined workspaces for any role they incorporate. In addition to that, users 
-can have their own workspaces. These are added to the permissions granted by roles.
+* **delete**: this element can be deleted
```
Comment (fashxp) [inline doc/22_Administration_of_Pimcore/07_Users_and_Roles.md]:
```suggestion
* **delete**: this element (and potential child elements) can be deleted
```

File: doc/22_Administration_of_Pimcore/07_Users_and_Roles.md [hunk 8]
```diff
@@ -55,62 +63,48 @@ The following list outlines what the different system permissions (available for
...
+* **rename**: this element name can be changed
```
Comment (fashxp) [inline doc/22_Administration_of_Pimcore/07_Users_and_Roles.md]:
```suggestion
* **rename**: this element's (and potential child elements') name can be changed
```

File: doc/22_Administration_of_Pimcore/07_Users_and_Roles.md [hunk 9]
```diff
@@ -55,62 +63,48 @@ The following list outlines what the different system permissions (available for
...
+* **settings**: this element's settings can be managed i.e. the settings tab is visible; the settings permission also the path and thereby the right to move the element in tree
```
Comment (fashxp) [inline doc/22_Administration_of_Pimcore/07_Users_and_Roles.md]:
```suggestion
* **settings**: this element's (and potential child elements') settings can be managed i.e. the settings tab is visible; the settings permission also the path and thereby the right to move the element in tree
```

**Review by fashxp — CHANGES_REQUESTED** [review]

**Review by fashxp — APPROVED** [review]

**Pull request merged**